test/jdk/java/lang/StrictMath/Log1pTests.java

```diff
@@ -209,6 +209,14 @@ static int testLog1p() {
             {0x1.7688bb5394bd3p325,     0x1.c34e8276daa48p7},
             {0x1.d42aea2878b45p328,     0x1.c7e96ee5c7f87p7},
             {0x1.249ad2594989p332,      0x1.cc845b54b54a6p7},
+
+            // Empirical worst-case points in other libraries with
+            // larger worst-case errors than FDLIBM
+            {-0x1.2bf183e0344b2p-2,     -0x1.62ebb44459d79p-2},
+            {-0x1.2bf32aaf122e2p-2,     -0x1.62ee0a3a4baf9p-2},
+            {-0x1.8000000000000p-53,    -0x1.8000000000001p-53},
+            {-0x1.2e496d25897ecp-2,     -0x1.663d81cb08f56p-2},
+            {-0x1.ffffffbaefe27p-2,     -0x1.62e42faa93817p-1},
         };
```

Comment:
I think there's another case
```suggestion
            {-0x1.5efad5491a79bp-1022,     -0x1.5efad5491a79bp-1022},
        };
            
```